Projects/App/Sources/Journeys/MainNavigationJourney.swift

```diff
@@ -98,7 +98,7 @@ class MainNavigationViewModel: ObservableObject {
                     let contractStore: ContractStore = globalPresentableStoreContainer.get()
                     await contractStore.sendAsync(.fetchContracts)
                     await checkForFeatureFlags()
-                    AnalyticsService().fetchAndSetUserId()
+                    try? await AnalyticsService().fetchAndSetUserId()
```

Comment:
Is it important to wait for this to execute after we log in?
I think putting this in the separated Task should be fix here